pkg/ssh/manifest.json.in

```diff
@@ -9,27 +9,27 @@
             "match": { "session": "private", "user": null, "host": null },
             "environ": [ "COCKPIT_SSH_CONNECT_TO_UNKNOWN_HOSTS=true",
                          "COCKPIT_PRIVATE_${channel}=${channel}" ],
-            "spawn": [ "@libexecdir@/cockpit-ssh", "${user}@${host}" ],
+            "spawn": [ "${libexecdir}/cockpit-ssh", "${user}@${host}" ],
```

Comment:
This OTOH should be fine -- cockpit-ssh is shipped in the cockpit-bridge package.

Reply:
... i'm sure glad I don't have to understand this stuff :+1: :smile: